backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/AbstractNumericDocValues.java

```diff
@@ -16,6 +16,9 @@
  * {@link DocIdSetIterator} APIs. This impl is safe to use for sorting and
  * aggregations, which only use {@link #advanceExact(int)} and
  * {@link #longValue()}.
+ * <p>
+ * Copied and adapted from {@code PUT THE ORIGINAL CLASS NAME HERE}
```

Comment:
I have removed "AbstractNumericDocValues.java ". I use a different solution. Everything is "DoubleMultiValuesSource" and "LongMultiValuesSource". This can sort and aggregate.